packages/liquidator/src/liquidationStrategy.js

```diff
@@ -168,15 +168,15 @@ module.exports = (
       // 4) Do nothing.
     }
 
-    // check if we should try to liquidate this position
-    if (shouldLiquidate({ tokensToLiquidate })) {
-      // We should liquidate the entire position using our capital - wdf
+    if (toBN(tokensToLiquidate).gt(toBN(0))) {
       return createLiquidationParams({
         sponsor: position.sponsor,
         tokensToLiquidate,
         currentBlockTime,
         maxCollateralPerToken
       });
+    } else {
+      return null;
```

Comment:
just a nit, not a big deal, but this is redundant. can just remove the whole else block and the function will return undefined. if you are explicitly checking for `null` then just return null without the else, since previous if block exits early.